implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -243,6 +245,14 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
     @ConfigProperty(name = "smallrye.jwt.verify.algorithm", defaultValue = "RS256")
     private Optional<SignatureAlgorithm> signatureAlgorithm;
 
+    /**
+     * Supported key format. By default a key can be in any of the supported formats:
+     * PEM key, PEM certificate, JWK key set or single JWK (possibly Base64URL-encoded).
+     */
+    @Inject
+    @ConfigProperty(name = "smallrye.jwt.verify.key-format", defaultValue = "ANY")
+    private Optional<KeyFormat> keyFormat;
```

Comment:
What are your thoughts on removing the `Optional`? Since a `defaultValue` is given, I'm not sure it adds anything. I know this is not the only property like this, so it's not unusual. From the standpoint of repetition, the default is given three times - here, in the `orElse` and also as the default in `JWTAuthContextInfo`.

Reply:
@MikeEdgar That was a polite way of suggesting the big redundancy is here, thanks :-). Yes, I keep copying and pasting. I'll start from fixing it from this PR